packages/craftcms-webpack/get-config.js

```diff
@@ -0,0 +1,309 @@
+  });
+};
+
+const applyDotEnv = ({ context, configName, currentConfigName }) => {
+  const isCurrentConfig = currentConfigName && configName === currentConfigName;
+  const envFilePath = getFirstExistingPath([
+    isCurrentConfig && path.join(context, ".env"),
+    path.join(rootPath, ".env"),
+  ].filter(Boolean));
+
+  if (envFilePath) {
+    return dotenv.config({ path: envFilePath });
+  }
+};
+
+const getDevServer = ({ context, watchPaths }) => {
+  // Find PHP asset bundles
+  let files = fs.readdirSync(context);
+  let assetBundleClasses = [];
+
+  for (let i = 0; i < files.length; i++) {
+    let filename = path.join(context, files[i]);
+    let stat = fs.lstatSync(filename);
+    if (!stat.isDirectory() && filename.indexOf(".php") > 0) {
+      let data = fs.readFileSync(filename);
+
+      if (data) {
+        let namespaceRegex = /namespace\s(.*?);/gs;
+        let classNameRegex = /class\s(.*?)\sextends/gs;
+        let m;
+        let n;
+        let namespace = null;
+        let className = null;
+
+        while ((m = namespaceRegex.exec(data)) !== null) {
+          // This is necessary to avoid infinite loops with zero-width matches
+          if (m.index === namespaceRegex.lastIndex) {
+            namespaceRegex.lastIndex++;
+          }
+
+          // The result can be accessed through the `m`-variable.
+          m.forEach((match, groupIndex) => {
+            if (groupIndex === 1) {
+              namespace = match;
+            }
+          });
+        }
+
+        while ((n = classNameRegex.exec(data)) !== null) {
+          // This is necessary to avoid infinite loops with zero-width matches
+          if (n.index === classNameRegex.lastIndex) {
+            classNameRegex.lastIndex++;
+          }
+
+          // The result can be accessed through the `m`-variable.
+          n.forEach((match, groupIndex) => {
+            if (groupIndex === 1) {
+              className = match;
+            }
+          });
+        }
+
+        if (namespace && className) {
+          assetBundleClasses.push(namespace + "\\" + className);
+        }
+      }
+    }
+  }
+
+  const https = process.env.DEV_SERVER_SSL_KEY && process.env.DEV_SERVER_SSL_CERT ? {
+    key: fs.readFileSync(process.env.DEV_SERVER_SSL_KEY),
+    cert: fs.readFileSync(process.env.DEV_SERVER_SSL_CERT),
+  } : false;
+  const host = process.env.DEV_SERVER_HOST || "localhost";
+  const port = process.env.DEV_SERVER_PORT || "8085";
+  const scheme = https ? "https" : "http";
+  const publicPath = process.env.DEV_SERVER_PUBLIC || `${scheme}://${host}:${port}/`;
+
+  return {
+    host,
+    https,
+    port,
+    allowedHosts: "all",
+    devMiddleware: {
+      publicPath,
+    },
+    headers: { "Access-Control-Allow-Origin": "*" },
+    hot: true,
+    client: {
+      overlay: {
+        errors: true,
+        warnings: false,
+      },
+    },
+    static: watchPaths.map(directory => ({
+      directory,
+      watch: true,
+    })),
```

Comment:
@nfourtythree I killed the `DEV_SERVER_CONTENT_BASE` env var…do we need it? Seems like you could just pass `watchPaths` for whatever you wanted to watch.